modules/create-policy-categories.adoc

```diff
@@ -1,18 +1,42 @@
 // Module included in the following assemblies:
 //
 // * operating/manage-security-policies.adoc
-:_module-type: PROCEDURE
+:_content-type: PROCEDURE
 [id="create-policy-categories_{context}"]
 = Creating policy categories
 
 You can create new policy categories from the system policies view.
 
+{product-title-short} version 3.74 provides a new method to create and manage policy categories in {product-title-managed} or in {product-title-short} if you have the PostgreSQL database (Technical Preview) enabled. All policy workflows other than policy creation remain unchanged when using this feature. For instructions on using this feature, see the second procedure in the following section.
+
+You can also configure policy categories by using the `PolicyCategoryService` API object. For more information, navigate to *Help* -> *API reference* in the {product-title-short} portal. 
+
+:FeatureName: PostgreSQL support
+include::snippets/technology-preview.adoc[]
+
 .Procedure
-. On the RHACS portal, navigate to *Platform Configuration* -> *Policies*.
+
+. On the {product-title-short} portal, navigate to *Platform Configuration* -> *Policy Management*.
 . From the *Policies* page, select the policy you want to edit.
 . Select *Actions* -> *Edit policy*.
-. Enter a new category name in the *Categories* field and then click *Create <category>*.
+. In the *Policy details* section, enter a new category name in the *Categories* field and then click *Create <category>*. 
++
+[NOTE]
+====
+This action is not available in {product-title-short} version 3.74 if you have the PostgreSQL database (Technical Preview) enabled. You must create new policy categories by using the procedure in the next section.
```

Comment:
This is a bit confusing on what the user should do. Again, I think you need to break these up into separate modules for whatever the scenarios are.

Or if you really need to keep them in the same procedure module, then you might be able to do a workaround where you do something like 

* To \<First scenario>
1. <steps for first scenario> 
2. ...
* To \<Second scenario>
1. <steps for second scenario> 
2. ...
* To \<Third scenario>
1. <steps for third scenario> 
2. ...

Reply:
Yeah, I think that would be better. Initially I was trying to keep them in the same module because the old way is going away when we migrate to PostgreSQL, and I envisioned just deleting the old procedure. However, since we have to support the old way for a while, I should just break out the new way now.